### PR TITLE
Highlight plaque with video, add a tv= param to filter mmb and rebirth plaques

### PR DESCRIPTION
**Pull request by incredibleray**

### CODE REVIEW

File: src/PlaqueCard.js [hunk 1]
```diff
@@ -16,24 +21,48 @@ function PlaqueCard() {
   
   useEffect(() => {
     if (elementRef.current != null) {
-      dispatch({type: "setHighlightPlaqueHeight", payload: elementRef.current.clientHeight});
+      dispatch({type: "setHighlightPlaqueHeight", payload: elementRef.current.clientWidth});
```
Comment (rjuang) [inline src/PlaqueCard.js:24]: Why is the height = clientWidth ?

File: src/PlaqueCard.js [hunk 2]
```diff
@@ -16,24 +21,48 @@ function PlaqueCard() {
...
+    }}     ref={elementRef} 
     >
-        <Card>
-        <ProgressiveImage src={highlightPlaque.src} placeholder={highlightPlaque.thumbnail}>
-  {(src) => <img src={src} style={{height:highlightPlaqueHeight.toString()+"px"}} />}
-</ProgressiveImage>
-        </Card>
-        </DialogContent>
+      <Stack direction="row" spacing={9}>
+        <div><img src={"https://plaquetv.blob.core.windows.net/plaques/perm-"+highlightPlaque.id+".png"} style={{height: "92vh"}} /></div>
```
Comment (rjuang) [inline src/PlaqueCard.js:48]: Are we still using generated images ?

File: src/plaques.json [hunk 3]
```diff
@@ -1,180 +1,143 @@
 [
-{"id": "0", "beneficiary": "xxx xxxxxxx", "sponsor": "xxxxxxx", "type": "mmb", "location": "WMT", "requestDate": "12/11/2022", "expiryDate": "12/12/2022", "searchable": true},
```
Comment (rjuang) [inline src/plaques.json:2]: Please revert this change. PII.

File: src/PlaqueCard.js (hunk 1 above)
Comment (incredibleray) [reply src/PlaqueCard.js:24]: I changed the names.

File: src/PlaqueCard.js (hunk 2 above)
Comment (incredibleray) [reply src/PlaqueCard.js:48]: can't just use the MmbPlaque class or PlaqueSelector class he
there are errors. So I used this stop-gap solution for now.
